blog/models.py

```diff
@@ -22,7 +30,7 @@ class Stori(models.Model):
     created_at = models.DateTimeField(auto_now_add=True)
     updated_at = models.DateTimeField(auto_now=True)
     status = models.IntegerField(choices=STATUS, default=0) #"""This here serves to indicate whether a stori has been published or not."""
-    #category 
+    category = models.ForeignKey(Category,on_delete=models.PROTECT)
```

Comment:
why not have this display from the category model but indicates all the available options as a list???

Reply:
i don't understand kindly expalin.

Reply:
If you add a category to the category model it will display as a list in the blog model.

Reply:
yeah exactly